env.c

```diff
@@ -282,9 +282,9 @@ void	_dmalloc_environ_process(const char *env_str, DMALLOC_PNT *addr_p,
     if (strncmp(this_p, LOGFILE_LABEL, len) == 0
 	&& *(this_p + len) == ASSIGNMENT_CHAR) {
       this_p += len + 1;
-      len = MIN(env_p - this_p, sizeof(log_path));
+      len = MIN(next_p - this_p, sizeof(log_path));
```

Comment:
it's better to change:
```
len = MIN(next_p - this_p + 1, sizeof(log_path));
```
and keep line 287 as before. Otherwise, log_path mayn't be terminated by '\0' correctly.
Here is the result like for "log=/path/to/log,...":
```
0000000000000...F
0123456789ABC...F
/path/to/log?...0
```
The value at the qustion mark is a random number.